README.md

```diff
@@ -187,7 +186,7 @@ $ source .env.sh
 You're good to go! Use the environment variables in application code like this
 
 ```
-apiKey := os.Getenv("OKTA_CLIENT_TOKEN")
+authProvider := os.Getenv("BCDA_AUTH_PROVIDER")
```

Comment:
Not to prolong this ticket, but should we remove this and just set the auth provide to "SSAS."

Reply:
You're right, thats not even a legitimate env var anymore.